static/flow/views/program-editor-block-palette.js

```diff
@@ -63,61 +63,9 @@ var ProgramEditorBlockPalette = function(options) {
 
         // Icon
         var menuIcon = $('<img class="menu-icon">');
-        if (type=="temperature") {
```

Comment:
Great!  I rushed this in so the PI/researchers could decide if they liked it, but never got a thumbs up from anyone but Lisa.  Hopefully, we don't have to rip these icons out and waste the work!